source/Camera/InfoUtilOther/GetCameraIntrinsics.m

```diff
@@ -0,0 +1,36 @@
+% Read the intrinsic calibration parameters of the Zivid camera (OpenCV model).
+
```

Comment:
Why is there comments here when non of the other files have?

Reply:
Legacy. The basic samples from SE have no comments or descriptions. The extra samples have. This one, for other languages, is an extra sample and so it has a description. We will add descriptions to the main samples in the SDK first and then spread over other languages.